chore(weave): Bump TS SDK version and update docs (#4612)
wandb/weave: docs/docs/guides/tracking/tracing.mdx

  </TabItem>
  <TabItem value="typescript" label="TypeScript">

>     Weave automatically tracks [calls to common LLM libraries](../integrations/index.md) like `openai`.

Reviewer: ```suggestion
    Weave automatically tracks [calls to common LLM libraries](../integrations/index.md), such as `openai`.
```

    });
    ```

>     Please also see [TypeScript SDK: Third-Party Integration Guide](../integrations/js.md) for complete setup guide for JS/TS projects.

Reviewer: ```suggestion
    For a complete setup guide for JS / TS projects, see the [TypeScript SDK: Third-Party Integration Guide](../integrations/js.md).
```
